dhis-2/dhis-api/src/main/java/org/hisp/dhis/reporttable/ReportTable.java

```diff
@@ -1110,4 +1130,21 @@ public ReportTable setGridTitle( String gridTitle )
         this.gridTitle = gridTitle;
         return this;
     }
+    
+    // -------------------------------------------------------------------------
+    // Temporary overriding getters so the Schema properties are read as mandatory.
+    // Required in order to enable backward support during the migration to the
+    // new Visualization API
+    // -------------------------------------------------------------------------
+
+    @Override
+    @JsonProperty
+    @JacksonXmlProperty( isAttribute = true )
+    @Description( "The name of this Object. Required and unique." )
+    @PropertyRange( min = 1 )
+    @Property( required = TRUE )

```

Comment:
This kind of code is needed so the e2e tests can pass, forcing this field as mandatory in the Schema.